third_party/protobuf/BUILD

```diff
@@ -7,3 +7,15 @@ java_library(
   ],
   licenses = ["permissive"],
 )
+
+native.genrule(
+    name = "protoc_exe",
+    # TODO(dino): Support other platforms.
+    srcs = [
+        "@protoc_linux_x86_64//file",
+    ],
+    executable = 1,
+    outs = ["protoc"],
+    cmd = "cat $(locations @protoc_linux_x86_64//file) > $@",
```

Comment:
This is the recommended workaround for Bazel's current lack of support for marking http_file rules as "executable = 1".

This just takes the file and copies it to a new file which is executable :P

@damienmg